starter-code/css/main.css

```diff
@@ -1,10 +1,133 @@
+/* General */
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font: 200 16px/1.5 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  margin: 0 auto;
+  /* box-sizing: border-box; */
+}
+
+h2 {
+  width: 320px;
+  font-size: 30px;
+  color: #143c5e;
+  margin-bottom: 20px;
+}
+
+p {
+  font-size: 20px;
+}
+
+.wrapper {
+  width: 1200px;
+  margin: 0 auto;
+}
+
+/* Blue background */
+
+.blue-back {
+  background-color: #4e7495;
+  height: 400px;
+  width: 100%;
+}
+
+/* Left Side  */
+.left-side {
+  margin-top: 150px;
+  width: 50%;
+  float: left;
+}
+.phones {
+  width: 420px;
+  float: right;
+  margin-right: 70px;
+}
+
+/* Right Side */
+.right-side {
+  margin-top: 180px;
+  width: 50%;
+  float: right;
```

Comment:
this would need to be moved over the left some more. You can always try playing around with padding or margin.